test/cbmc/proofs/getFileContextFromJob/getFileContextFromJob_harness.c

```diff
@@ -84,7 +86,7 @@ DocParseErr_t parseJobDoc( const JsonDocParam_t * pJsonExpectedParams,
 
 void getFileContextFromJob_harness()
 {
-    OtaFileContext_t * pFileContext;
+    OtaFileContext_t * pFileContext = &fileContext;
```

Comment:
Why not get rid of this previously unused variable instead of now assigning it a value?